internal/services/oracle/cloud_vm_cluster_resource.go

```diff
@@ -229,6 +230,13 @@ func (CloudVmClusterResource) Arguments() map[string]*pluginsdk.Schema {
 			ForceNew: true,
 		},
 
+		"ocid": {
+			Type:     pluginsdk.TypeString,
+			Optional: true,
+			Computed: true,
+			ForceNew: true,
+		},
```

Comment:
Validation?

Reply:
The OCID isn't something we accept as input, it's something that is generated by our backend and we want customers to be able to reference e.g:
After creation: `azurerm_oracle_cloud_vm_cluster.example.ocid`
Is there a better way to define the property in the above case?

Reply:
Then it shouldn't be marked as `Optional` or `ForceNew` and should be added to the `Attributes` instead of the `Arguments`

Reply:
Thanks - I've moved it!